### ISSUE TITLE
Not Able to Configure Which iOS Architectures Are Built (When Using AppCenter) 

### ISSUE THREAD

**Issue by bobber205** [react-native-community/cli#252]
## Environment

  React Native Environment Info:
    System:
      OS: macOS High Sierra 10.13.6
      CPU: (4) x64 Intel(R) Core(TM) i5-5287U CPU @ 2.90GHz
      Memory: 40.06 MB / 8.00 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 11.6.0 - /usr/local/bin/node
      Yarn: 1.13.0 - /usr/local/bin/yarn
      npm: 6.5.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
      Android SDK:
        API Levels: 26, 27, 28
        Build Tools: 26.0.2, 27.0.3, 28.0.3
        System Images: android-27 | Google APIs Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 3.3 AI-182.5107.16.33.5314842
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.6.3 => 16.6.3
      react-native: 0.58.6 => 0.58.6
## Description

(Note: Please don't complain I am not on the latest and greatest of React. I literally just upgraded 2 weeks ago and I'm dealing with this nasty build issue I can't possible introduce another giant wrench in the works. This happened on both 57 and now 58.)

When I try to build my react native project for iOS on AppCenter it's trying to build for the old arch called `armv7`.  A library I am using is not compatible with `armv7`.

```
PhaseScriptExecution Install\ Third\ Party /Users/vsts/Library/Developer/Xcode/DerivedData/legit-fphykrdjnudzmkcnvzkdevlhgesk/Build/Intermediates.noindex/ArchiveIntermediates/legit/IntermediateBuildFilesPath/React.build/Release-iphoneos/double-conversion.build/Script-190EE32F1E6A43DE00A8543A.sh
    cd /Users/vsts/agent/2.148.1/work/1/s/node_modules/react-native/React
    export ACTION=install
    export AD_HOC_CODE_SIGNING_ALLOWED=NO
    export ALTERNATE_GROUP=staff
    export ALTERNATE_MODE=u+w,go-w,a+rX
    export ALTERNATE_OWNER=vsts
    export ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES=NO
    export ALWAYS_SEARCH_USER_PATHS=NO
    export ALWAYS_USE_SEPARATE_HEADERMAPS=NO
    export APPLE_INTERNAL_DEVELOPER_DIR=/AppleInternal/Developer
    export APPLE_INTERNAL_DIR=/AppleInternal
    export APPLE_INTERNAL_DOCUMENTATION_DIR=/AppleInternal/Documentation
    export APPLE_INTERNAL_LIBRARY_DIR=/AppleInternal/Library
    export APPLE_INTERNAL_TOOLS=/AppleInternal/Developer/Tools
    export APPLICATION_EXTENSION_API_ONLY=NO
    export APPLY_RULES_IN_COPY_FILES=NO
    export ARCHS="armv7 arm64"
    export ARCHS_STANDARD="armv7 arm64"
    export ARCHS_STANDARD_32_64_BIT="armv7 arm64"
    export ARCHS_STANDARD_32_BIT=armv7
    export ARCHS_STANDARD_64_BIT=arm64
    export ARCHS_STANDARD_INCLUDING_64_BIT="armv7 arm64"
    export ARCHS_UNIVERSAL_IPHONE_OS="armv7 arm64"
```

That's the first set of references to armv7.

Here is the failure line:

```
The following build commands failed:
	CompileC /Users/vsts/Library/Developer/Xcode/DerivedData/legit-fphykrdjnudzmkcnvzkdevlhgesk/Build/Intermediates.noindex/ArchiveIntermediates/legit/IntermediateBuildFilesPath/Pods.build/Release-iphoneos/RNImageCropPicker.build/Objects-normal/armv7/ImageCropPicker.o /Users/vsts/agent/2.148.1/work/1/s/node_modules/react-native-image-crop-picker/ios/src/ImageCropPicker.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
##[error]Error: /usr/bin/xcodebuild failed with return code: 65
```

My VALID_ARCHITECTURES variable in my xcode settings doesn't include it.

![image](https://user-images.githubusercontent.com/5841107/54639882-a56e9080-4a4b-11e9-91e0-d80ef859fb40.png)

As far as I can tell this is coming from the React internal files? How can I get rid of armv7?

**Comment by thymikee** [comment]
We'll need to investigate this. Can you provide a minimal repro we can run and see what happens? 

Just to let you know, this probably won't be fixed in 0.58, as this repo contains the CLI for 0.59+ (unless the eventual fix will be easy to port an cherrypick to 0.58)

**Comment by thymikee** [comment]
Closing due to lack of activity
